### PR TITLE
togt2022badder

### PR DESCRIPTION
**Pull request by thesofakillers**
## Project Name

togt2022badder

## Compiled PDF

The pdf failed to compile. See #50 

## Metadata Checklist

- [x] `title` field exists and it starts with `[Re] `.
- [x] `authors` contain author information with affiliations.
- [x] [ORCID id](https://orcid.org/) is provided for authors.
- [x] One contact author is added using `*`.
- [x] Github `url` is provided in `code` section.
- [x] [SWH identifier](https://archive.softwareheritage.org/save/) is provided in `swh`.
- [x] `replication` section is populated with the necessary information about the paper you reproduced.
- [x] `abstract` field is populated with your paper abstract (including double quotes)
- [x] `domain` should contain "ML Reproducibility Challenge 2021"
- [x] `review` section contains the url to the OpenReview forum of your paper.
- [x] Fields `contributors`, `dates`, `article` and `journal` are kept blank, as those will be populated by the Area Chairs / ReScience editors.

## Build Errors

As reported in #50, see the following pastebin(s):

- [https://pastebin.com/iw5U7xSL](https://pastebin.com/iw5U7xSL)

### CODE REVIEW

File: 2021/togt2022badder/journal/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+review:
+  - url: https://openreview.net/forum?id=HcIxA3Mm2CF
+
+contributors:
+  - name: Koustuv Sinha
+    orcid:
+    role: editor
+  - name: Anonymous Reviewers
+    orcid:
+    role: reviewer
```
Comment (koustuvsinha) [inline 2021/togt2022badder/journal/metadata.yaml:113]: Hi @thesofakillers , unfortunately the build script fails as the ReScience template expects two entries minimum for the role "reviewer". Please add a blank name block as we have provided in the template following this line.

```
- name:
    orcid:
    role: reviewer
```

**Comment by thesofakillers** [comment]
PDF now compiles albeit with glitches, see [here]
(https://github.com/thesofakillers/MLRC/blob/main/2021/togt2022badder/journal/article.pdf). Namely:
- citations were not found
  - i've tried copying the openreview/bibliography.bib file to journal/bibliography.bib to no avail
- figure/section references not working
- reviewer names overlapping with first page
These issues seem similar to #48.

Please let me know how to proceed with these issues. Thank you.

**Comment by koustuvsinha** [comment]
I have reproduced this issue locally. The problem stems from importing `natbib` package, as ReScience already takes care of it (https://github.com/ReScience/MLRC/pull/1#issuecomment-817266184). 

- Please remove the import of `natbib` package, and
- Copy the contents of your own `openreview/bibliography.bib` file to `journal/bibliography.bib` file.

**Comment by thesofakillers** [comment]
Thanks for all the help @koustuvsinha. Everything seems to work now.

**Comment by thesofakillers** [comment]
noticed some minor glitches/niggles, had to fix them and add some commits. Sorry

**Comment by thesofakillers** [comment]
I've noticed some people are adding the missing "[Re]" to their title in accepted.bib, should we do that too? So far I've only added that to metadata.yaml and our pdf

**Comment by koustuvsinha** [comment]
Hi,

It is not necessary, as we do not use the accepted.bib in any other situation - it was only used to create the folders.

Best,

Koustuv


On Apr 30, 2022, at 2:03 PM, Giulio Starace ***@***.***> wrote:

﻿

I've noticed some people are adding the missing "[Re]" to their title in accepted.bib, should we do that too? So far I've only added that to metadata.yaml and our pdf

**Pull request merged**